tests/conftest.py

```diff
@@ -64,10 +64,10 @@ def inner(  # pylint: disable=too-many-arguments,missing-docstring,useless-suppr
         engine,
         func_workchain_name,
         engine_kwargs,
-        xtol,
-        ftol,
-        x_exact,
-        f_exact,
+        xtol=None,
```

Comment:
I don't completely understand why these need to be made optional: they are the exact value and tolerance of the expected result, not parameters passed to the optimization engine.
AFAICT, we can have a known result for the optimization even if we don't pass tolerances to the engine itself -- we can make the tolerances "large enough" to ensure we don't get false test failures.

Reply:
Ok, I will revert the changes and I will put a huge number for the tolerances in my case.

Reply:
> a huge number

I mean, only needs to be such that the algorithm can't reasonably be outside of this bound.

I'd say, take the highest deviation you find in e.g. 10 runs, multiply it by ~10, that should be big enough. And still give us some indication that the algorithm went to the right place.